core/validation_schema.py

```diff
@@ -144,17 +144,16 @@
             "Project ID",
             "Start Date",
             "Completion Date",
-            "Current Project Delivery Stage",
+            # Current Project Delivery Status is sometimes nullable so has its own specific validation
             "Project Delivery Status",
             # Leading Factor of Delay is sometimes nullable so has its own specific validation
             "Project Adjustment Request Status",
             "Delivery (RAG)",
             "Spend (RAG)",
             "Risk (RAG)",
             "Commentary on Status and RAG Ratings",
-            "Most Important Upcoming Comms Milestone",
-            "Date of Most Important Upcoming Comms Milestone (e.g. Dec-22)",
-            # TODO: Added all fields for Round 4 (was previously only Project ID)
+            # Most Important Upcoming Comms Milestone is sometimes nullable so has its own specific validation
+            # This also applies to Date of Most Important Upcoming Comms Milestone (e.g. Dec-22)
```

Comment:
I think this is another use case for splitting dataframes up in a previous stage. We could have `ProjectProgress` and `ProjectProgressComplete` - and then we could use the schema validation as usual? Not one for now just a thought

Reply:
I'm not sure how straightforward that would be. Splitting DataFrames makes a lot of sense when we're retaining the tables as they original appear in the spreadsheet (like footfall and non-footfall outcomes), but splitting up a 'natural' DataFrame into two artificial ones based on how our validation works might be less clear than the bespoke validation. One to look at I suppose

Reply:
Hmmm maybe you're right. We need to try to move away from this kind of specific validation though and have a repeatable, configurable way of doing it.. Moving to a more simple spreadsheet built by us will help but I think this kind of scenario could arise then as well potentially.